### DISCUSSION TITLE
About OTLP  with skywalking

### DISCUSSION THREAD

**Discussion by shalk** [apache/skywalking#12377] [General]
Hi, everyone 

from https://github.com/apache/skywalking/pull/11909  Skywalking have support OTLP.

trace, log and metric have been supported.
https://skywalking.apache.org/docs/main/next/en/setup/backend/otlp-trace/
https://skywalking.apache.org/docs/main/next/en/setup/backend/log-otlp/#opentelemetry-logging-format
https://skywalking.apache.org/docs/main/next/en/setup/backend/opentelemetry-receiver/#opentelemetry-metrics-format

In the detail,  trace will convert to zipkin trace , so the ui is on zipkin' ui. 

I found "NOTICE, Zipkin trace would not be analyzed like SkyWalking native trace format." https://skywalking.apache.org/docs/main/next/en/setup/backend/zipkin-trace/

So is any plan to support convert otel trace to SkyWalking native trace format ?  I can create issue on this.

**Comment by wu-sheng** [comment]
No, it can't. SkyWalking uses STAM to analysis traces, so, we have specific requirements in the data. 

https://skywalking.apache.org/docs/main/latest/en/papers/stam/

If that is possible, there is no point we have to build all agents. SkyWalking's performance relies on its unique design.
